ArcGISRuntimeSDKQt_CppSamples/Layers/AddDynamicEntityLayer/AddDynamicEntityLayer.qml

```diff
@@ -0,0 +1,141 @@
+// [WriteFile Name=AddDynamicEntityLayer, Category=Layers]
+// [Legal]
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick
+import QtQuick.Controls
+import Esri.Samples
+
+Item {
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the map etc. and supply the view
+    AddDynamicEntityLayerSample {
+        id: model
+        mapView: view
+        onConnectionStatusChanged: {
+            // Enable switch if data stream is successfully disconnected or connected
+            statusSwitch.enabled = (["Disconnected", "Connected"].indexOf(model.connectionStatus) !== -1)
+        }
+    }
+
+    Rectangle {
+        id: dynamicEntityLayerOptions
+        width: (300 < parent.width - (anchors.margins * 2)) ? 300 : parent.width - (anchors.margins * 2)
+        height: column.height + (column.anchors.margins * 2)
```

Comment:
not critical, but these are pretty complicated. Perhaps there is a simpler way to achieve this